### PR TITLE
Validate $ANON

### PR DESCRIPTION
**Pull request by Pandybary**
Validate Anon

# Validate [$ANON](https://solscan.io/token/4PEMPJXiYd4THpNuaFTf2zB8jKYJJPoEN86vdvKnjv9R)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://twitter.com/AnonCoinSol/status/1657374688304046082
- Coingecko/ CMC URL (If available): https://www.coingecko.com/en/coins/{id}

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [ ] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.

### CODE REVIEW

**Comment by Pandybary** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.
 
Mint address: 
4PEMPJXiYd4THpNuaFTf2zB8jKYJJPoEN86vdvKnjv9R
Name: ANON
Symbol: $ANON
Token Decimals: 7
Token icon: https://arweave.net/kPNmtSROIdowJp2fUp1A02vqEwD-9bduQTbgiZugsd0
On-chain metadata matches
No duplication on Jupiter's strict list

**Comment by kkrkishore** [comment]
Mint address:
4PEMPJXiYd4THpNuaFTf2zB8jKYJJPoEN86vdvKnjv9R
Name: ANON
Symbol: $ANON
Token Decimals: 7
Token icon: https://arweave.net/hSyqZL4l_8wpk291raYcTZlBs0hNzHncklBQWALOB8E
On-chain metadata matches
No duplication on Jupiter's strict list

File: validated-tokens.csv [hunk 1]
```diff
@@ -726,4 +726,5 @@ KING,KING,9noXzpXnkyEcKF3AeXqUHTdR59V5uvrRBUZ9bwfQwxeq,9,https://i.imgur.com/IQ4
 Poor token,POOR,2mqi8iLJf9WaVRHd7nhRkqvCafxfb3Us7wshaJR73ykS,5,https://shdw-drive.genesysgo.net/AZgm3Vdo1QLf8bKUjyearDC9NygpRHmmtzPaBNQcF4jt/IMG_1640.JPG,true
 Solcasino Token,SCS,SCSuPPNUSypLBsV4darsrYNg4ANPgaGhKhsA3GmMyjz,5,https://files.solcasino.io/logo.png,true
 Sunrise gSOL,GSOL,gso1xA56hacfgTHTF4F7wN5r4jbnJsKh99vR595uybA,9,https://arweave.net/46QNhUyat533ptVC2myiNVb5BvSP7PAFzM5ILCZTrGU,true
-Access Protocol,ACS,5MAYDfq5yxtudAhtfyuMBuHZjgAbaS9tbEyEQYAhDS5y,6,https://ap-staging.fra1.digitaloceanspaces.com/1663691449945,true
\ No newline at end of file
+Access Protocol,ACS,5MAYDfq5yxtudAhtfyuMBuHZjgAbaS9tbEyEQYAhDS5y,6,https://ap-staging.fra1.digitaloceanspaces.com/1663691449945,true
+Anon,$ANON,4PEMPJXiYd4THpNuaFTf2zB8jKYJJPoEN86vdvKnjv9R,7,https://arweave.net/kPNmtSROIdowJp2fUp1A02vqEwD-9bduQTbgiZugsd0,true
```
Comment (Timoon21) [inline validated-tokens.csv:730]:
```suggestion
ANON,$ANON,4PEMPJXiYd4THpNuaFTf2zB8jKYJJPoEN86vdvKnjv9R,7,https://arweave.net/hSyqZL4l_8wpk291raYcTZlBs0hNzHncklBQWALOB8E,true
```

**Review by Timoon21 — CHANGES_REQUESTED** [review]
- [ ] On-chain metadata matches the PR

**Review by Timoon21 — DISMISSED** [review]
- [x] On-chain metadata matches the PR
- [x] Token address matches the tweet
- [x] No duplication on Jupiter's strict list
- [x] No duplication on CMC/CoinGecko
- [x] Community validation

**Review by Timoon21 — APPROVED** [review]

**Comment by MGAdON** [comment]
All of the data is correct and matches.

Mint address:
4PEMPJXiYd4THpNuaFTf2zB8jKYJJPoEN86vdvKnjv9R
Name: ANON
Symbol: $ANON
Token Decimals: 7
Token icon: https://arweave.net/hSyqZL4l_8wpk291raYcTZlBs0hNzHncklBQWALOB8E
On-chain metadata matches
No duplication on Jupiter's strict list

**Pull request merged**